### DISCUSSION TITLE
VJudge 查询失败

### DISCUSSION THREAD

**Discussion by ChillsLove** [Liu233w/acm-statistics#3019] [Bug reports]
提示 Cannot process vjudge data, body: {"error":"Please login first"}

**Comment by ChillsLove** [comment]
vj是不是需要自己部署后，配置自己的账号密码才能进行查询，在ojhunt网站上不能自定义查询自己的数据。如果是这样的话，这个讨论我还是关了吧

**Reply by Liu233w** [reply]
抱歉我最近没有看GitHub。

正常情况下VJ是可以直接用的，因为我的网站上面有设置账户。现在出这个问题有可能是VJ的界面或者API变了。等我DEBUG一下吧。

**Reply by Liu233w** [reply]
Hi, 已经修复了
